frontend/server/src/DAO/Contests.php

```diff
@@ -458,6 +460,14 @@ final public static function getContestsParticipating(
             ON
                 pi.problemset_id = p.problemset_id AND
                 pi.identity_id = ?
+            INNER JOIN
+                ACLs
+            ON
+                Contests.acl_id = ACLs.acl_id
+            INNER JOIN
+                Identities
```

Comment:
```suggestion
                Identities AS organizer
```

siempre intentamos tener un alias para las tablas _excepto_ `Contests` en este archivo en las consultas donde se usa `$columns`. normalmente se usan las iniciales de la tabla, pero aquí se puede usar `owner` porque nada más `i` puede no ser suficientemente legible.

igual en el resto del archivo.

Reply:
Owner o Organizer?

Reply:
errr... organizer ^^;; por consistencia con el nombre de la columna resultante.

Reply:
Ok, va